install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -258,7 +258,8 @@ global:
   sds:
     enabled: false
     udsPath: ""
-    enableTokenMount: false
+    useK8sSATrustworthyJwt: false
+    useK8sSANormalJwt: false
```

Comment:
Nit again: so a normal Jwt is not to be trusted?

Maybe it's verified Jwt?

Reply:
the normal k8s sa jwt could be verified against k8s api server, just the trustworthy just has more claims like expired time.